DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: pacta.portfolio.allocate
 Title: pacta.portfolio.allocate
-Version: 0.0.2.9000
+Version: 0.0.2.9001
```

Comment:
```suggestion
Version: 0.0.2.9000
```

Reply:
Doesn't bumping version make sense, given that there's an incompatible signature change on the functions?

Reply:
It's never been done before and I don't intend to do it in the future

Reply:
My two cents is that this is NIT either way. 

To me, the version `X.Y.Z.9000` indicates that this is a dev version, and may not be compatible with other  `X.Y.Z.9000` versions. 

but ever since https://github.com/RMI-PACTA/actions/blob/main/.github/workflows/R-semver-check.yml , I have personally started bumping the `9000` version on every commit, mainly to satisfy the check. 

So I would suggest just leaving this to @cjyetman's discretion here (as the maintainer).

Reply:
I'm fine with not bumping version here.

The reason I wrote the semver check and like using it is that it is a very clear way to identify not just that something is a dev version, but to specify which dev version is required. 

It's really nice to have something like this:

```
Imports:
  pacta.portfolio.allocate (>= 0.0.2.9001),
```
https://github.com/RMI-PACTA/workflow.pacta/blob/d4edcbba0291bbf56550952796de14fb88f497c4/DESCRIPTION#L32

to flag that there's inconsistencies (and probably errors) if we try to use the older (`.9000`) version.